src/Components/Navbar.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from "react";
+import "./Navbar.css";
+import Logo from "./assets/Logo.png";
+import searchButton from "./assets/search-button.png";
+export default function Navbar() {
+  // const handleClick = (e) => {
+  //   let button = document.getElementsByClassName("buttons");
+  //   for (let i = 0; i < button.length; i++) {
+  //     if (button[i].hasAttribute("id", "active")) {
+  //       button[i].removeAttribute("id", "active");
+  //     }
+  //   }
+  //   e.target.setAttribute("id", "active");
+  // };
+  const [imgNo, setImgNo] = useState(0);
+
+  useEffect(() => {
+    setInterval(() => {
```

Comment:
Hey I am not sure why you are using `setInterval`, but if it is required, please also use `clearInterval` in useEffect return. To know more, read this -> https://www.codementor.io/@damianpereira/how-to-use-clearinterval-inside-react-s-useeffect-and-why-it-is-important-1si7mztjlk